mne_neuron/cell.py

```diff
@@ -299,7 +298,9 @@ def parconnect_from_src(self, gid_presyn, nc_dict, postsyn):
         nc : instance of h.NetCon
             A network connection object.
         """
-        nc = self.pc.gid_connect(gid_presyn, postsyn)
+        from mne_neuron import sim
```

Comment:
did you forget to add the file sim.py to the diff?

Reply:
Oops!